chia/util/api_decorators.py

```diff
@@ -7,6 +7,13 @@
 log = logging.getLogger(__name__)
 
 
+BYTES_REQUIRED = "bytes_required"
+PEER_REQUIRED = "peer_required"
+EXECUTE_TASK = "execute_task"
+REPLY_TYPE = "reply_type"
```

Comment:
Small naming thing: reply_type is pretty generic,  but I understand not wanting to make it too long.

Maybe `message_reply_type` or `protocol_reply_type`?

I only mention it because it's an object attribute - as a local variable name it's great.